admin/nodes/views.py

```diff
@@ -118,6 +118,17 @@ def post(self, request, *args, **kwargs):
 
         return redirect('nodes:node', guid=kwargs.get('guid'))
 
+
+class NodeUpdateModerationStateView(View):
+    def post(self, request, *args, **kwargs):
+        guid = kwargs.get('guid')
+        node = AbstractNode.load(guid)
```

Comment:
This should be good since it AbstractNode uses `GuidMixin`. No need to update this to `.load_referent()`

Reply:
As expected, no issue found after merging latest `develop`.